dashboard/client/pages/_app.tsx

```diff
@@ -1,29 +1,30 @@
 import App, { Container } from 'next/app'
 import React from 'react'
 
-import Layout from '../components/layout'
 import '../styles/reset.scss'
 
 class MyApp extends App {
   static async getInitialProps({ Component, ctx }) {
     let pageProps = {}
+    try {
+      if (Component.getInitialProps) {
+        pageProps = await Component.getInitialProps(ctx)
+      }
 
-    if (Component.getInitialProps) {
-      pageProps = await Component.getInitialProps(ctx)
+      return { pageProps }
+    } catch (e) {
+      throw new Error(e)
```

Comment:
`console.error(e.message)`, instead of throwing a new Error 🙊

Reply:
it don let me